### PR TITLE
Update to DHCP monitoring in document

### PR DESCRIPTION
**Pull request by efuaakum**
Update to make docs consistent across repositories

Co-authored-by: Joshua-Luke Bevan <joshua.bevan@madetech.com>

### CODE REVIEW

File: documentation/monitoring.md [hunk 1]
```diff
@@ -1,12 +1,42 @@
...
+- [NLB ProccessBytes](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-cloudwatch-metrics.html)
+- [UnHealthyHostCount](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-cloudwatch-metrics.html)
+- [ECS MemoryUtilization and CPUUtilization](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cloudwatch-metrics.html)
+- [RDS ReadIOPS, WriteIOPS and CPUUtilization](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage.Metrics)
+
+## Custom DHCP Metrics
+
+![Custom DHCP Metrics](./images/monitoring/custom_panel.png)
+
+The Custom section displays all DHCP metrics output by [Kea](https://cloudsmith.io/~isc/repos/kea-1-8/packages/)
```
Comment (efuaakum) [inline documentation/monitoring.md:40]:
```suggestion
The custom section displays all DHCP metrics outputted by [Kea](https://cloudsmith.io/~isc/repos/kea-1-8/packages/) as well as a subset of [error and debug messages](https://kea.readthedocs.io/en/kea-1.8.0/kea-messages.html).

- CONFIGURATION RELOAD SUCCESSFUL
- CONFIGURATION RELOAD FAILED
- CONFIGURATION RELOAD SUCCESSFUL
- CONFIGURATION RELOAD FAILED
- ALLOC_ENGINE_V4_ALLOC_ERROR
- ALLOC_ENGINE_V4_ALLOC_FAIL
- ALLOC_ENGINE_V4_ALLOC_FAIL_CLASSES
```

File: documentation/monitoring.md [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+
+- `OK` is a sign that the system is operating normally
+- `Pending` indicates that the system may be either recovering or erroring
+- `Alerting` shows that the system needs attention
+
+## AWS Metrics
+
+![aws](./images/monitoring/aws_panel.png)
+
+The AWS section displays all the relevant metrics to AWS. These include:
```
Comment (neilkidd) [inline documentation/monitoring.md:28]:
```suggestion
This section displays all the relevant AWS metrics. These include:
```

File: documentation/monitoring.md [hunk 3]
```diff
@@ -1,12 +1,48 @@
...
+- [NLB ProccessBytes](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-cloudwatch-metrics.html)
+- [UnHealthyHostCount](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-cloudwatch-metrics.html)
+- [ECS MemoryUtilization and CPUUtilization](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cloudwatch-metrics.html)
+- [RDS ReadIOPS, WriteIOPS and CPUUtilization](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage.Metrics)
+
+## Custom DHCP Metrics
+
+![Custom DHCP Metrics](./images/monitoring/custom_panel.png)
+
+The custom section displays all DHCP metrics outputted by [Kea](https://cloudsmith.io/~isc/repos/kea-1-8/packages/) as well as a subset of [error and debug messages](https://kea.readthedocs.io/en/kea-1.8.0/kea-messages.html).
```
Comment (neilkidd) [inline documentation/monitoring.md:40]:
```suggestion
The custom section displays all DHCP metrics produced by [Kea](https://cloudsmith.io/~isc/repos/kea-1-8/packages/) as well as a subset of [error and debug messages](https://kea.readthedocs.io/en/kea-1.8.0/kea-messages.html).
```

File: documentation/monitoring.md [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+
+- CONFIGURATION RELOAD SUCCESSFUL
+- CONFIGURATION RELOAD FAILED
+- ALLOC_ENGINE_V4_ALLOC_ERROR
+- ALLOC_ENGINE_V4_ALLOC_FAIL
+- ALLOC_ENGINE_V4_ALLOC_FAIL_CLASSES
+
+The dashboards monitor [subnet and network metrics](https://kea.readthedocs.io/en/kea-1.8.1/arm/stats.html) including the rate of change of DORA operations (Discovery, Offer, Request and Acknowledge).
```
Comment (neilkidd) [inline documentation/monitoring.md:48]:
```suggestion
The dashboards monitor [subnet and network metrics](https://kea.readthedocs.io/en/kea-1.8.1/arm/stats.html) including the rate of change for DORA operations (Discovery, Offer, Request and Acknowledge) and Not Acknowledge (NAK) operations.
```

**Review by neilkidd — APPROVED** [review]
A couple of suggestions :) Otherwise LGTM

**Pull request merged**